modules/swagger-compat-spec-parser/src/main/java/io/swagger/parser/SwaggerCompatConverter.java

```diff
@@ -489,7 +509,21 @@ public ApiDeclaration readDeclaration(String input, MessageBuilder messages, Lis
                 String json = RemoteUrl.urlToString(input, auths);
                 jsonNode = Json.mapper().readTree(json);
             } else {
-                jsonNode = Json.mapper().readTree(new java.io.File(input));
+                final String fileScheme = "file:";
```

Comment:
Any reason for duplicate code ? Can't we refactor this code into function

Reply:
I took this exact code from https://github.com/swagger-api/swagger-parser/blob/master/modules/swagger-parser/src/main/java/io/swagger/parser/Swagger20Parser.java#L46
Maybe it makes sense to refactor this code too?